src/datahike_jdbc/core.clj

```diff
@@ -1,9 +1,15 @@
 (ns datahike-jdbc.core
-  (:require [datahike.store :refer [empty-store delete-store connect-store default-config config-spec release-store]]
+  (:require [datahike.store :refer [empty-store delete-store connect-store default-config config-spec release-store store-identity]]
             [datahike.config :refer [map-from-env]]
             [konserve-jdbc.core :as k]
             [clojure.spec.alpha :as s]))
 
+(defmethod store-identity :jdbc [store-config]
+  (let [{:keys [jdbcUrl dbtype host port dbname]} store-config]
+    (if jdbcUrl
+      [:jdbc jdbcUrl]
```

Comment:
You wanna mention it in the README?

Reply:
This is internal, it is just providing a vector that uniquely determines the underlying store for a database. Maybe I am missing where to put it in the README.

Reply:
ok, got it